docs/components/sidecar.md

```diff
@@ -25,6 +25,7 @@ Prometheus servers connected to the Thanos cluster via the sidecar are subject t
 
   * [Querier](./query.md) to filter out store APIs to touch during query requests.
   * Many object storage readers like [compactor](./compact.md) and [store gateway](./store.md) which groups the blocks by Prometheus source. Each produced TSDB block by Prometheus is labelled with external label by sidecar before upload to object storage.
+  We support use env value to specify it, matching the regular expression `\$\(([a-zA-Z_0-9]+)\)`.
```

Comment:
Not sure if we need to overwhelm used with the exact regular expression. 

Maybe just add mention the format to the sentence about env replacing already?
The part in `Reloader` section
```Thanos sidecar can watch `--reloader.config-file=CONFIG_FILE` configuration file,  environment variables found in there, and produce generated config in `--reloader.config-envsubst-file=OUT_CONFIG_FILE` file.```
to
```Thanos sidecar can watch `--reloader.config-file=CONFIG_FILE` configuration file, replace environment variables found in there in `$(VARIABLE)` format, and produce generated config in `--reloader.config-envsubst-file=OUT_CONFIG_FILE` file.```

What do you think?

Reply:
done.

Reply:
@FUSAKLA now promethues support use env in config? Seems i misunderstand.

Reply:
No, Prometheus does not support it, this is done by the Thanos sidecar reloader. 
Sorry maybe my formulation is not well understandable.